### DISCUSSION TITLE
Anthropic plugin

### DISCUSSION THREAD

**Discussion by vasicstefan94** [Ironclad/rivet#382] [General]
Hello!
How can I use the Anthropic chat node with rivet-node. I'm trying to call runGraphInFile but I'm getting the error : Error: Unknown node type: chatAnthropic. I have upgraded to 1.16.0. 

Thanks in advance!

**Comment by abrenneke** [comment]
Looks like we answered on discord but for search results, the easiest way is to use the `globalRivetNodeRegistry`:

```ts
import { globalRivetNodeRegistry } from '@ironclad/rivet-node';
import { plugins as rivetPlugins } from '@ironclad/rivet-core';

globalRivetNodeRegistry.registerPlugin(rivetPlugins.anthropic);
```
